CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ option(WITH_GTKDOC "Enables libdnf GTK-Doc HTML documentation" ON)
 option(WITH_HTML "Enables hawkey HTML generation" ON)
 option(WITH_MAN "Enables hawkey man page generation" ON)
 option(WITH_ZCHUNK "Build with zchunk support" ON)
+option(WITH_REPOCONFIG_DAEMON "Build dnf-repoconfig-daemon" ON)
```

Comment:
I'd probably turn the default to OFF.
It's something additional and it also requires C++17 while the current libdnf requires C++11.